apicapi/config.py

```diff
@@ -434,7 +434,8 @@ def create_switch_dictionary():
                 switch_dict[switch_id]['pod_id'] = port[0]
                 continue
             hosts = host_list.split(',')
-            hosts = map(lambda a: a.decode('string_escape'), hosts)
+            import codecs
```

Comment:
Imports should be at the top of the module, and appear in alphabetic order.

Reply:
oh yes!